docs/assets/stylesheets/custom.css

```diff
@@ -0,0 +1,32 @@
+.md-search__form {
+    border-radius: 24px;
```

Comment:
Use rem units

Reply:
Will have to explore the theme to check root element